python/housinginsights/ingestion/load_data.py

```diff
@@ -0,0 +1,458 @@
+"""
+Modulel loads our flat file data into the Postgres database. It will rebuild the
+database of choice with sample or real data by first deleting any data in the
+repository, and then re-load the data into the repository.
+
+If you want to rebuild the data with actual data (i.e. what is in manifest.csv
+instead of manifest.csv), run the same command without ‘sample’ at the end.
+
+Notes:
+ - manifest.csv has every flat file that needs to be loaded (i.e. CSV's we have
+ downloaded).
+ - other scripts can get data that is available from APIs, so manifest won't
+ reflect all the data we are including.
+ - meta.json provides meta information about our *SQL* tables. Note that
+ because multiple CSV's can go into the same table (i.e. two different versions
+ of the same data), there will be more rows in the CSV than there are 'tables'
+ in the json.
+"""
+
+import sys
+import os
+import logging
+import argparse
+import json
+from sqlalchemy import Table, Column, Integer, String, MetaData
+
+# Needed to make relative package imports when running this file as a script
+# (i.e. for testing purposes).
+# Read why here: https://www.blog.pythonlibrary.org/2016/03/01/python-101-all
+# -about-imports/
+
+PYTHON_PATH = os.path.abspath(os.path.join(os.path.dirname(__file__),
+                                           os.pardir, os.pardir))
+sys.path.append(PYTHON_PATH)
+
+from housinginsights.tools import dbtools
+
+from housinginsights.ingestion import DataReader, ManifestReader
+from housinginsights.ingestion import CSVWriter, DataReader
+from housinginsights.ingestion import HISql, TableWritingError
+from housinginsights.ingestion import functions as ingestionfunctions
+
+# configuration: see /logs/example-logging.py for usage examples
```

Comment:
Don't actually want this here - move it into the if name == main statement. If its here but the code is imported from another file (like from /scripts or from the API server) we'll end up with duplicated log statements (this comment from experience)